proxy/src/avatar.rs

```diff
@@ -209,9 +226,10 @@ fn hash(input: &str) -> u64 {
     hash
 }
 
+#[allow(clippy::float_cmp, clippy::non_ascii_literal)]
 #[cfg(test)]
 mod test {
-    use super::*;
+    use super::{generate_color, generate_emoji, hash, Avatar, Color, Emoji, Usage};
```

Comment:
Since we're only importing from the current file, it seems redundant to restate everything here, and just adds additional maintenance.

Reply:
The reason these are explicitly imported is for a reader to easier understand the origin of used function or type. Again it might be the case now that only the same file is imported but this tends to change over time and established good hygiene from the get-go will avoid harder to maintain code down the road.

Reply:
Yeah, I also like this approach to explicitly import stuff.